rpm/util-linux.spec

```diff
@@ -198,6 +199,24 @@ Obsoletes: %{name}-docs
 %description doc
 Man and info pages for %{name}.
 
+%package systemd-fstrim
+Summary:   Systemd unit files for %{name}
+Group:     System
+Requires:  %{name} = %{version}-%{release}
+Requires(post): systemd
+
+%description systemd-fstrim
+Systemd unit files for fstrim.
+
+%package systemd-uuidd
+Summary:   Systemd unit files for %{name}
+Group:     System
+Requires:  %{name} = %{version}-%{release}
+Requires:  uuidd
+Requires(post): systemd
+
+%description systemd-uuidd
+Systemd unit files for uuidd
```

Comment:
The same issue with rpmlint here.

Reply:
O.K., [done](https://github.com/sailfishos/util-linux/pull/2/commits/8f0e9bfe3f4f802c07394c892b3006dfc04afb12).

Reply:
Nevertheless, still [please do implement cosmetic changes, if you have any additional ones: This PR is explicitly marked as "maintainers may alter it" etc. [...]](https://github.com/sailfishos/util-linux/pull/2#discussion_r699330306)

Reply:
@Olf0 Its normal that if you want to contribute that you have to follow the guidelines of the platform.
In our case see here the specific for packaging: https://sailfishos.org/wiki/Software_Packaging